### PR TITLE
Add Services to Load test

### PR DESCRIPTION
**Pull request by gmarek**

### CODE REVIEW

**Comment by wojtek-t** [comment]
This is exactly what I expected - thanks!

One question before I lgtm it - did you look how it affects performance of this test in e.g. 500-node kubemark and 1000-kubemark? I think we should know it before lgtm-ing it.

**Comment by gmarek** [comment]
Sadly no. I'll leave it running on a screen today.

**Comment by gmarek** [comment]
This change does not work on Kubemark (seems that Kubemark doesn't support services):
```
33 Apr 20 22:29:52.321: INFO: Unexpected error occurred: Internal error occurred: failed to allocate a serviceIP: range is full
```

I'll look into it tomorrow.

**Comment by gmarek** [comment]
Added service-ip-range flag which seems to help. Kubemark should read all flags from the config files though:/

**Comment by wojtek-t** [comment]
OK - so let me ask again - what' the impact on the results in large clusters?

**Comment by gmarek** [comment]
I run it on Kubemark-500 and it's not great:
```
Apr 21 00:59:59.698: INFO: WARNING Top latency metric: {Resource:replicationcontrollers Verb:LIST Latency:{Perc50:253.848ms Perc90:888.761ms Perc99:1.043219s}}
Apr 21 00:59:59.698: INFO: WARNING Top latency metric: {Resource:endpoints Verb:LIST Latency:{Perc50:132.795ms Perc90:414.913ms Perc99:743.732ms}}
Apr 21 00:59:59.698: INFO: Top latency metric: {Resource:services Verb:LIST Latency:{Perc50:59.754ms Perc90:176.534ms Perc99:379.878ms}}
Apr 21 00:59:59.698: INFO: Top latency metric: {Resource:nodes Verb:LIST Latency:{Perc50:59.378ms Perc90:118.155ms Perc99:214.108ms}}
Apr 21 00:59:59.698: INFO: Top latency metric: {Resource:pods Verb:LIST Latency:{Perc50:17.467ms Perc90:40.444ms Perc99:138.545ms}}
```

And the suite itself run much longer:
```
Ginkgo ran 1 suite in 5h56m14.208499974s
```

**Comment by gmarek** [comment]
Added an env variable to gate service creation - I didn't want to push the flag to the main e2e binary:/

File: test/e2e/load.go [hunk 1]
```diff
@@ -36,14 +38,16 @@ const (
...
-	bigRCGroupName    = "load-test-big-rc"
+	smallRCGroupName  = "load-small-rc"
+	mediumRCGroupName = "load-medium-rc"
+	bigRCGroupName    = "load-big-rc"
 	smallRCBatchSize  = 30
 	mediumRCBatchSize = 5
 	bigRCBatchSize    = 1
 )
 
+var createServices = os.Getenv("CREATE_SERVICES")
```
Comment (wojtek-t) [inline test/e2e/load.go]: I would prefer avoid having it as a global variable. Can you please move it to the test?

Comment (wojtek-t) [inline test/e2e/load.go]: I mean next to where it actually is used.

**Comment by wojtek-t** [comment]
Minor nit - other than that lgtm

File: test/e2e/load.go (hunk 1 above)
Comment (gmarek) [inline test/e2e/load.go]: I wanted it to be better visible, but OK.

**Comment by gmarek** [comment]
@wojtek-t PTAL

**Comment by wojtek-t** [comment]
LGTM

**Comment by gmarek** [comment]
@k8s-bot unit test this issue: #24628

**Comment by gmarek** [comment]
Travis is really unhappy today...

**Comment by gmarek** [comment]
Reopening to make travis run again...

**Pull request merged**